tests/builtins/functions/test_process.py

```diff
@@ -29,6 +29,7 @@ def add_event_info(_dict):
 @pytest.fixture(autouse=True)
 def process_env(workflow):
     workflow_prefix = workflow["stateMachineArn"].rsplit(":", 1)[0] + ":"
+    os.environ["CIRRUS_EVENT_DB_AND_TABLE"] = "event-db-1|event-table-1"
```

Comment:
I don't think you need this here if it is in the builtins `env` fixture.

Reply:
yeah, this was leftover from when I was trying to figure out why the env var wasn't being propagated correctly.  removed.